lib/id3c/cli/command/refresh_materialized_view.py

```diff
@@ -0,0 +1,36 @@
+"""
+Refresh materialized view in ID3C.
+"""
+import click
+import logging
+from id3c.cli import cli
+from id3c.cli.command import with_database_session
+from id3c.db.session import DatabaseSession
+
+
+LOG = logging.getLogger(__name__)
+
+
+@cli.command("refresh-materialized-view")
+@with_database_session
+
+@click.argument("schema-name",
+    metavar = "<schema>",
+    nargs = 1)
+@click.argument("view-name",
+    metavar = "<view>",
+    nargs = 1)
+
+def refresh_materialized_view(schema_name, view_name, db: DatabaseSession):
+    """
+    Refresh materialized view <schema>.<view> in ID3C.
+    """
+
+    LOG.info(f"Refreshing materialized view «{schema_name}.{view_name}»")
+
+    db.cursor("refresh materialized view").execute("""
+        select refresh_materialized_view(%s, %s)
+    """, (schema_name, view_name ))
```

Comment:
By explicitly naming the cursor, this creates a server-side cursor. This ends up suppressing any errors. For example, `id3c refresh-materialized-view doesnotexist notevenalittle` reports success. I'm not entirely sure why, but I assume this is because Pg defers raising the exception, or maybe executing the query at all, until the cursor is first read. Since we never read from the cursor, nothing happens. Removing the argument to `db.cursor()` means we use a client-side cursor, which works as expected.

Reply:
Thank you for catching this!